webapp/src/components/CurrencyNumpad.js

```diff
@@ -20,6 +22,18 @@ export default function CurrencyNumpad({ onCentsChange, whole, currency, cents }
         </div>
       </div>
       <Numpad cents={cents} currency={currency} onNumberClick={handleChange} />
+      <Form.Text className="d-block text-secondary text-end">
```

Comment:
Helpful idea but I would remove it- users don't really need to know it. If they exceed one of the bounds, we tell them. We pick the values we have because they're unlikely to exceed the bounds.